.landscaper/component-descriptor.yaml

```diff
@@ -0,0 +1,12 @@
+component:
```

Comment:
this file is not needed as is generated by the gardener ci during the build.

For local development I suggest to generate is on the fly with the component-cli .e.g 
```
component-cli component-archive resources add "${CA_PATH}" "$SOURCE_PATH"/.landscaper/resources.yaml \
--commponent-name github.com/gardener/gardener-extension-provider-aws \
--component-version v0.0.1 \
--repo-url=..
```

Reply:
we have removed it